ice40/pack.cc

```diff
@@ -373,11 +375,10 @@ static void pack_constants(Context *ctx)
         ctx->cells[gnd_cell->name] = std::move(gnd_cell);
         ctx->nets[gnd_net->name] = std::move(gnd_net);
     }
-    // Vcc cell always inserted for now, as it may be needed during carry legalisation (TODO: trim later if actually
-    // never used?)
-    ctx->cells[vcc_cell->name] = std::move(vcc_cell);
-    ctx->nets[vcc_net->name] = std::move(vcc_net);
-
+    if (vcc_used) {
```

Comment:
This change could cause carry packing to break, as it assumes the presence of the global Vcc net. Probably would be better to not add the cell and net if it already exists (which I presume is the real purpose of this change).